### PR TITLE
Run integration tests with Eclipse in komodo

### PR DESCRIPTION
**Pull request by berland**
Ported from the Jenkins setup. The contents of res-testdata/Equinor is no longer in use and not included in the port.

**Issue**
Resolves missing running of integration tests with Eclipse in Komodo nightly testing.

**Approach**
Port Jenkins seteup.

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Updated documentation
- [ ] Ensured new behaviour is tested

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

File: ci/testkomodo.sh [hunk 1]
```diff
@@ -25,6 +25,7 @@ start_tests () {
     # Allow xvfb to find a new server
     pushd ${CI_TEST_ROOT}/tests
     xvfb-run -s "-screen 0 640x480x24" --auto-servernum python -m \
-    pytest -m "not requires_window_manager"
+        pytest -m "not requires_window_manager"
```
Comment (xjules) [inline ci/testkomodo.sh:28]: There is an extra indentation or ?

File: ci/testkomodo.sh [hunk 2]
```diff
@@ -25,6 +25,7 @@ start_tests () {
...
+    pytest -m "requires_eclipse" --eclipse-simulator
```
Comment (berland) [inline ci/testkomodo.sh:29]: The  part `-m "requires_eclipse"` does the subset selection of 17 eclipse related tests.  These 17 are skipped by default, but the option `--eclipse-simulator` turns them on.

Comment (xjules) [reply ci/testkomodo.sh:29]: Right, now it makes sense :+1:

**Comment by berland** [comment]
Before merge: Check if this requires /global/bin needs to be in PATH (which it is not by default for action runners)

**Comment by berland** [comment]
> Before merge: Check if this requires /global/bin needs to be in PATH (which it is not by default for action runners)

Check done. Removing `/global../ from `$PATH` and then running pytest manually goes fine. ✔️

**Review by oysteoh — APPROVED** [review]

**Pull request merged**